### DISCUSSION TITLE
Size /pixel Import from file

### DISCUSSION THREAD

**Discussion by jimklma** [ImageMagick/ImageMagick#4152] [General]
Hi
I'. New on this forum I search the forum but could't find this issue
Q If I want to resize 1000 images in the same folder but every image has his own size. This size and image names are in a Excel file. Could I connect IM to this file and change the imagesize by reading yhe excel file?
For example:
Excel column A reads the image name
Column B the height in pixels
Column C the width in pixels
Column D the image folder path
How can IM read the Excel file and run is this possdible?

Thanks in advance
Greenhorn Jim

**Comment by snibgo** [comment]
What version of IM, on what platform?

IM cannot read Excel files.

From inside Excel, you can export the data as a CSV file. This is a text file, with values separated by commas. Then a shell script can read that CSV file, parse the filename, width, height and folder, and call IM wth those parameters.
